### PR TITLE
Release notes for 3.0.0

### LINKED ISSUE #4048: 3.0.0 release
I'd like to release 3.0.0 soon to get cleanups and fixes to users.

Any objections to doing that once remaining issues in the milestone https://github.com/translate/translate/milestone/25 are addressed?

### PR DESCRIPTION
**Pull request by nijel**
Fixes #4048

### CODE REVIEW

**Comment by nijel** [comment]
CI will most likely see some failure due to https://github.com/pypa/virtualenv/issues/1857

File: docs/releases/3.0.0.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+This release contains improvements and bug fixes.
+
+
+Changes
+=======
+
+Formats and Converters
+----------------------
+
+- l20n
```
Comment (unho) [inline docs/releases/3.0.0.rst:15]: This section is usually sorted with the most widely used formats first. This should probably be the last one.

File: docs/releases/3.0.0.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+- PHP
+
+    - Add support for Laravel plurals
+    - Improve round trip of some statements
+
+- XLIFF 
+
+    - Support non numeric ids on plurals in poxliff
+
```
Comment (unho) [inline docs/releases/3.0.0.rst:49]: These notes are missing the removal of deprecated code like `segment` module, and also https://github.com/translate/translate/pull/4045#pullrequestreview-428678746

Comment (nijel) [reply docs/releases/3.0.0.rst:50]: Yes, I forgot these. Adding them now.

**Pull request merged**